src/bci_build/package/mariadb-client/README.md.j2

```diff
@@ -0,0 +1,27 @@
+# {{ image.pretty_name }} container image
+
+## Description
+
+MariaDB is an open-source, multi-threaded, relational database management system. It's a backward-compatible branch of the MySQL Community Server that provides a drop-in replacement for MySQL.
+
+This image contains the [MariaDB client](https://mariadb.com/kb/en/mariadb-command-line-client/) and utilities.
```

Comment:
Should I list all utilities included in the image? Or can we assume that users will know what comes in this container?

I had no clue. First I looked at what was in `/usr/bin` and then I looked at what binaries were installed by mariadb-client package.

Reply:
I think it makes sense to list the shipped client binaries

Reply:
The issue is that the binaries start with `mariadb` and `mysql`, not sure if I list both, or just MariaDB version.

This is why I felt leaving it out would be better.

Reply:
The `mysql*` variants are only symlinks to the respective `mariadb*` binary. Hence it's imho sufficient to mention the `mariadb*` versions

Reply:
I added mariadb only in a new section.